index.js

```diff
@@ -1,16 +1,21 @@
 const path = require('path');
 const { existsSync } = require('fs');
 const relative = require('require-relative');
-const { version } = require('svelte/package.json');
 const { createFilter } = require('rollup-pluginutils');
 const { encode, decode } = require('sourcemap-codec');
 
-const major_version = +version[0];
 const pkg_export_errors = new Set();
 
-const { compile, preprocess } = major_version >= 3
-	? require('svelte/compiler.js')
-	: require('svelte');
+// future: breaks on v10.0
+const toMajor = str => Number(str[0]);
```

Comment:
Can we snake case this to `to_major`? And the limitation of breaking at v10 seems pretty remote at this point, but it seems easy to address, with a `str.split('.')[0]` or a `str.match(/^[^.]+/)[0]` or whatever.

Reply:
The beauty of `6.x` is that we'll now never get there 😆 But ya I can adjust